Assets/Scripts/CustomExtensions.cs

```diff
@@ -9,9 +9,19 @@ public static void AddToEnemySpawnPointList(this List<List<EnemySpawnPoint>> spa
         {
             spawnPointList[spawnPointToAdd.areaIndex].Add(spawnPointToAdd);
         } 
-        catch(ArgumentOutOfRangeException e)
+        catch(ArgumentOutOfRangeException)
         {
             spawnPointList.Add(new List<EnemySpawnPoint>() { spawnPointToAdd });
         }
     }
+
+    public static List<EnemySpawnPoint> ToValidSpawnPoints(this List<EnemySpawnPoint> spawnPointList)
+    {
+        return spawnPointList.FindAll(
+            delegate(EnemySpawnPoint spawnPoint)
```

Comment:
Good luck explaining this one to others :D

Reply:
I will, it's easy, but just looks a bit weird at first 😂